parsec/core/gui/enrollment_widget.py

```diff
@@ -44,7 +44,7 @@ def __init__(
         self.setupUi(self)
 
         self.dialog: GreyedDialog[AcceptCheckInfoWidget] | None = None
-        self.label_waiting.hide()
```

Comment:
What was this `label_waiting` ?

Reply:
Before we re-did the onboarding process, the spinner was contained into a QLabel named `label_waiting`. This label is now wrapped into a QWidget named `widget_waiting`. `label_waiting` still exists so the code doesn't crash, but it's not doing what it's supposed to do.

Reply:
ok, thanks